### ISSUE TITLE
RFE: Add generic way to specify device nodes to attach to each pod

### ISSUE THREAD

**Issue by dustymabe** [kubernetes-sigs/cri-o#1821]
**Description**

RFE: Add generic way to specify device nodes to attach to each pod that is deployed via cri-o. The idea here is that you want something like `/dev/fuse` or `/dev/kvm` mounted into pods so that users can use them without having to use `privelged: true` or `hostPath:`. 

This would obsolete the need for things like [oci-kvm-hook](https://github.com/stefwalter/oci-kvm-hook)

related: https://github.com/kubernetes/kubernetes/issues/5607

**Comment by dustymabe** [comment]
cc @stefwalter

**Comment by mrunalp** [comment]
Yeah, we can add this to crio.conf to inject some devices by default.

**Comment by pcahyna** [comment]
related: https://github.com/kubernetes/kubernetes/issues/60748 ?
